src/main/java/core/basesyntax/ColorSupplier.java

```diff
@@ -1,7 +1,10 @@
 package core.basesyntax;
 
+import java.util.Random;
+
 public class ColorSupplier {
-    public String getRandomColor() {
-        return null;
+    public Colors getRandomColor() {
+        int color = new Random().nextInt(Colors.values().length);
```

Comment:
probably it is better to move new Random() initialization out of this method - like in your Lottery class